.github/workflows/ci.yml

```diff
@@ -26,7 +26,5 @@ jobs:
         run: biome ci .
       - name: Build SDK
         run: pnpm build-sdk
-      - name: Check types
-        run: pnpm check-types
```

Comment:
Since type-checks are performed during the next build in the deploy
process, there's no need to perform them in CI, so we will remove them.